setup.py

```diff
@@ -43,6 +29,24 @@ def get_version(rel_path):
     return "{}.{}.{}".format(major_version, minor_version, patch_version)
 
 
+def get_install_requires():
+    install_requires = [
+        'neo4j',
+        'praatio~=5.0',
+        'textgrid',
+        'conch_sounds',
+        'librosa',
+        'influxdb',
+        'tqdm',
+        'requests',
+        'scipy~=1.12.0',
+        'setuptools' # only added to work with praatio
+    ]
+    if os.name == 'nt':
+        install_requires.append('pywin32')
+    return install_requires
+
+
 if __name__ == '__main__':
     setup(name='polyglotdb',
           version=get_version("polyglotdb/__init__.py"),
```

Comment:
I think it'd be ideal to swap over to [setuptools_scm](https://github.com/pypa/setuptools_scm) for generating versioning based off of git tags/releases to mitigate the risk that the code-based version diverges from the actual ground truth versioning from github releases.